fuel-merkle/src/common/path_iterator.rs

```diff
@@ -145,14 +145,13 @@ impl<T> Iterator for PathIter<T>
 where
     T: ParentNode + Clone,
     T::Key: Path + Clone,
```

Comment:
Hmm, maybe we also can remove those `Clone`?=)

Reply:
Yes, we can=)

Reply:
Good catch!